1.regression_modelling/scripts/0.split_data.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from pycytominer import feature_select
+
+
+# ## Load in normalized profiles and concat into one dataframe
+
+# In[2]:
+
+
+# path to big drive where data is located (large)
+drive_path = pathlib.Path("../../../../media/18tbdrive/")
+
+# set path to folder in image profiling repo with the normalized profiles
+norm_path = pathlib.Path(
+    drive_path / "Github_Repositories/nuclear_speckle_image_profiling/4.preprocess_features/data/single_cell_profiles"
+).resolve(strict=True)
+
+# load all normalized parquet files
```

Comment:
Is each file a plate?

Reply:
Yes, each file relates to a "slide" or plate in our terminology.